### PR TITLE
Provides a helper which directly creates potentially long running que…

### PR DESCRIPTION
**Pull request by andyHa**
…ries on the statistics warehouse.

(Note, the queries are not made long running by the system but rather the framework will not report
these queries, as these are expected to be "slower" when crunching billions of rows).

### CODE REVIEW

File: src/main/java/sirius/biz/analytics/events/EventRecorder.java [hunk 1]
```diff
@@ -135,6 +136,27 @@ public Database getDatabase() {
         return database;
     }
 
+    /**
+     * Directly creates a new quey on the datastore used for events.
+     * <p>
+     * This is a Clickhouse database and the given query is automatically marked as potentiall long running to
```
Comment (meg1502) [inline src/main/java/sirius/biz/analytics/events/EventRecorder.java:142]:
```suggestion
     * This is a Clickhouse database and the given query is automatically marked as potentially long running to
```

File: src/main/java/sirius/biz/analytics/events/EventRecorder.java [hunk 2]
```diff
@@ -135,6 +136,27 @@ public Database getDatabase() {
         return database;
     }
 
+    /**
+     * Directly creates a new quey on the datastore used for events.
```
Comment (bwiedmann) [inline src/main/java/sirius/biz/analytics/events/EventRecorder.java:140]:
```suggestion
     * Directly creates a new query on the datastore used for events.
```

File: src/main/java/sirius/biz/analytics/events/EventRecorder.java [hunk 3]
```diff
@@ -135,6 +136,27 @@ public Database getDatabase() {
...
+     * <p>
+     * This is a Clickhouse database and the given query is automatically marked as potentially long running to
+     * suppress warnings if the query takes longer than a usual SQL query (which is kind of expected for large
+     * data warehouses).
+     *
+     * @param sql the SQL to execute. Note that this should be a constant string as parameters can later be passed in
+     *            using {@link SQLQuery#set(String, Object)}
```
Comment (bwiedmann) [inline src/main/java/sirius/biz/analytics/events/EventRecorder.java:147]: Codestyle?

Comment (sabieber) [reply src/main/java/sirius/biz/analytics/events/EventRecorder.java:147]: Einrückung passt hier glaub

**Pull request merged**